### PR TITLE
fix checkstyle and enable checkstyle plugin

### LINKED ISSUE #330: [Feature] fix checkstyle and enable checkstyle plugin
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-hugegraph-toolchain/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Feature Description (功能描述)

fix checkstyle and enable checkstyle plugin

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by simon824**
closed #330

### CODE REVIEW

**Review by coderzc — APPROVED** [review]

File: hugegraph-tools/src/main/java/com/baidu/hugegraph/cmd/SubCommands.java [hunk 1]
```diff
@@ -1058,17 +1060,17 @@ public List<HugeType> convert(String value) {
                     hugeTypes.add(HugeType.valueOf(type.toUpperCase()));
                 } catch (IllegalArgumentException e) {
                     throw new ParameterException(String.format(
-                              "Invalid --type '%s', valid value is 'all' or " +
-                              "combination of 'vertex,edge,vertex_label," +
-                              "edge_label,property_key,index_label'", type));
+                            "Invalid --type '%s', valid value is 'all' or " +
```
Comment (javeme) [inline hugegraph-tools/src/main/java/com/baidu/hugegraph/cmd/SubCommands.java:1063]: can we keep the origin style? prefer to keep consistent with server code--change the code style specification: cancel the strict restriction on indentation

File: hugegraph-tools/src/main/java/com/baidu/hugegraph/formatter/kgdumper/DumpKGFormatter.java [hunk 2]
```diff
@@ -101,6 +101,7 @@ private String dumpEntity(JsonVertex vertex)
                     region.add(((String) edge.getTarget()).split(":", 2)[1]);
                     regionWeight.add(props.get("weight").toString());
                     break;
+                default:
```
Comment (javeme) [inline hugegraph-tools/src/main/java/com/baidu/hugegraph/formatter/kgdumper/DumpKGFormatter.java:104]: also add break

File: hugegraph-tools/src/main/java/com/baidu/hugegraph/cmd/SubCommands.java [hunk 3]
```diff
@@ -1058,17 +1060,17 @@ public List<HugeType> convert(String value) {
...
+                            "edge_label,property_key,index_label'", type));
                 }
             }
             return hugeTypes;
         }
     }
 
     public static class AuthHugeTypeConverter
-                  implements IStringConverter<List<HugeType>> {
+            implements IStringConverter<List<HugeType>> {
```
Comment (javeme) [inline hugegraph-tools/src/main/java/com/baidu/hugegraph/cmd/SubCommands.java:1073]: can we keep the origin style?

File: hugegraph-tools/src/main/java/com/baidu/hugegraph/formatter/kgdumper/SignFS64.java [hunk 4]
```diff
@@ -115,7 +137,7 @@ private static String longBytes2String(ByteBuffer buffer) {
     }
 
     public static String createSignFs64(String text, String encode)
-                                        throws UnsupportedEncodingException {
+            throws UnsupportedEncodingException {
```
Comment (javeme) [inline hugegraph-tools/src/main/java/com/baidu/hugegraph/formatter/kgdumper/SignFS64.java:140]: ditto

**Review by javeme — COMMENTED** [review]
we need this feature, it may bring a lot of convenience

File: checkstyle.xml [hunk 5]
```diff
@@ -2,86 +2,216 @@
 <!DOCTYPE module PUBLIC
```
Comment (simon824) [inline checkstyle.xml:2]: This file copy from https://github.com/apache/incubator-hugegraph/blob/master/style/checkstyle.xml , to keep consistent with hg server

**Comment by simon824** [comment]
hi @imbajin, please take a look at this pr when free ,thanks.

File: hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java [hunk 6]
```diff
@@ -56,14 +56,22 @@ public static void main(String[] args) {
...
+        Vertex laurence = graph.addVertex(T.LABEL,
+                                          "person", "name", "laurence Fishburne", "born", 1961);
+        Vertex hugo = graph.addVertex(T.LABEL,
+                                      "person", "name", "hugo Weaving", "born", 1960);
+        Vertex lillyW = graph.addVertex(T.LABEL,
+                                        "person", "name", "Lilly Wachowski", "born", 1967);
+        Vertex lanaW = graph.addVertex(T.LABEL,
+                                       "person", "name", "Lana Wachowski", "born", 1965);
+        Vertex joelS = graph.addVertex(T.LABEL,
+                                       "person", "name", "Joel Silver", "born", 1952);
```
Comment (imbajin) [inline hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java:74]: seems no need to wrap new line in 59~74

File: hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java [hunk 7]
```diff
@@ -73,10 +81,12 @@ public static void main(String[] args) {
...
         joelS.addEdge("PRODUCED", theMatrix, "score", 10);
 
-        Vertex emil = graph.addVertex(T.label, "person", "name", "emil Eifrem", "born", 1978);
+        Vertex emil = graph.addVertex(T.LABEL, "person", "name", "emil Eifrem", "born", 1978);
         emil.addEdge("ACTED_IN", theMatrix, "roles", "emil");
 
-        Vertex theMatrixReloaded = graph.addVertex(T.label, "movie", "title", "The Matrix Reloaded", "released", 2003);
+        Vertex theMatrixReloaded = graph.addVertex(T.LABEL,
+                                                   "movie", "title",
+                                                   "The Matrix Reloaded", "released", 2003);
```
Comment (imbajin) [inline hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java:89]: Is it a rule? Seems a little strange?

File: hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java (hunk 6 above)
Comment (simon824) [reply hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java:74]: LineLength longer than 100 here

**Review by javeme — APPROVED** [review]

File: hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java (hunk 6 above)
Comment (imbajin) [reply hugegraph-client/src/main/java/com/baidu/hugegraph/example/MovieExample.java:74]: OK, keep them

**Review by imbajin — APPROVED** [review]

File: hugegraph-client/src/main/java/com/baidu/hugegraph/structure/constant/T.java [hunk 8]
```diff
@@ -21,6 +21,6 @@
 
 public class T {
 
-    public static final String id = "id";
-    public static final String label = "label";
+    public static final String ID = "id";
+    public static final String LABEL = "label";
```
Comment (coderzc) [inline hugegraph-client/src/main/java/com/baidu/hugegraph/structure/constant/T.java:25]: Why change this?

Comment (javeme) [reply hugegraph-client/src/main/java/com/baidu/hugegraph/structure/constant/T.java:25]: maybe to adapt code style for the const var

**Pull request merged**